src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  box-sizing: border-box;
+  --hover_color: #00acdc;
```

Comment:
Create a variable with root selector
https://developer.mozilla.org/en-US/docs/Web/CSS/var

Reply:
still not fixed